utils.py

```diff
@@ -189,6 +189,35 @@ def get_predictions(output):
     indices = indices.view(bs,h,w) # bs x h x w
     return indices
 
+# This function is written by Ying @UIUC RSim group. In this case, we have assumed the batch size is 1.
+def getCoordinates(predict):
+    pupilPixel = np.where(predict[0].cpu().numpy() == 3)
+    xMin = min(pupilPixel[0])
+    xMax = max(pupilPixel[0])
+    yMin = min(pupilPixel[1])
+    yMax = max(pupilPixel[1])
+    delta = (xMax - xMin) - (yMax - yMin)
+    
+    if delta >= 0:
```

Comment:
An explanation of this math in a comment or a link to where you found the math for this would be helpful.